.concourse/post-publish.yaml

```diff
@@ -0,0 +1,346 @@
+
```

Comment:
Could you name this file `post_publish.yaml`? We are using snake-case for all the files in the project, except the ones used by code generation that match a name that contains another case style. E.g. BOSH instance groups and job names.